keyboards/keebio/bfo9000/keymaps/vladkvit/layers.json

```diff
@@ -0,0 +1 @@
+[["KC_NO", "KC_NO", "KC_ESC", "KC_F1", "KC_F2", "KC_F3", "KC_F4", "KC_F5", "KC_NO", "KC_NO", "KC_F6", "KC_F7", "KC_F8", "KC_F9", "KC_F10", "KC_F11", "KC_F12", "KC_NO", "KC_NO", "KC_NO", "KC_GRV", "KC_1", "KC_2", "KC_3", "KC_4", "KC_5", "KC_NO", "KC_NO", "KC_6", "KC_7", "KC_8", "KC_9", "KC_0", "KC_MINS", "KC_EQL", "KC_BSPC", "KC_NO", "KC_NO", "KC_TAB", "KC_Q", "KC_W", "KC_E", "KC_R", "KC_T", "KC_NO", "KC_NO", "KC_Y", "KC_U", "KC_I", "KC_O", "KC_P", "KC_LBRC", "KC_RBRC", "KC_BSLS", "KC_NO", "KC_NO", "KC_NO", "KC_A", "KC_S", "KC_D", "KC_F", "KC_G", "KC_NO", "KC_NO", "KC_H", "KC_J", "KC_K", "KC_L", "KC_SCLN", "KC_QUOT", "KC_ENT", "KC_NO", "KC_NO", "KC_NO", "KC_LSFT", "KC_Z", "KC_X", "KC_C", "KC_V", "KC_B", "KC_NO", "KC_NO", "KC_N", "KC_M", "KC_COMM", "KC_DOT", "KC_SLSH", "KC_RSFT", "KC_UP", "KC_NO", "KC_NO", "KC_NO", "KC_LCTL", "KC_NO", "KC_LGUI", "KC_LALT", "KC_NO", "KC_SPC", "MO(1)", "KC_NO", "KC_SPC", "KC_NO", "KC_RALT", "KC_LGUI", "KC_RCTL", "KC_LEFT", "KC_DOWN", "KC_RGHT"], ["KC_NO", "KC_NO", "KC_ESC", "KC_F1", "KC_F2", "KC_F3", "KC_F4", "KC_F5", "KC_NO", "KC_NO", "KC_F6", "KC_F7", "KC_F8", "KC_F9", "KC_F10", "KC_F11", "KC_F12", "KC_NO", "KC_NO", "KC_NO", "KC_GRV", "KC_1", "KC_2", "KC_3", "KC_4", "KC_5", "KC_NO", "KC_NO", "KC_6", "KC_7", "KC_8", "KC_9", "KC_0", "KC_MINS", "KC_EQL", "KC_BSPC", "KC_NO", "KC_NO", "KC_TAB", "KC_Q", "KC_W", "KC_E", "KC_R", "KC_T", "KC_NO", "KC_NO", "KC_Y", "KC_INS", "KC_HOME", "KC_PGUP", "KC_P", "KC_LBRC", "KC_RBRC", "KC_BSLS", "KC_NO", "KC_NO", "KC_NO", "KC_A", "KC_S", "KC_D", "KC_F", "KC_G", "KC_NO", "KC_NO", "KC_H", "KC_DEL", "KC_END", "KC_PGDN", "KC_SCLN", "KC_QUOT", "KC_ENT", "KC_NO", "KC_NO", "KC_NO", "KC_LSFT", "KC_Z", "KC_X", "KC_C", "KC_V", "KC_B", "KC_NO", "KC_NO", "KC_N", "KC_M", "KC_COMM", "KC_DOT", "KC_SLSH", "KC_RSFT", "KC_UP", "KC_NO", "KC_NO", "KC_NO", "KC_LCTL", "KC_LGUI", "KC_LALT", "KC_NO", "KC_NO", "KC_SPC", "KC_TRNS", "KC_NO", "KC_SPC", "KC_NO", "KC_RALT", "KC_LGUI", "KC_RCTL", "KC_LEFT", "KC_DOWN", "KC_RGHT"]]
```

Comment:
This file actually doesn't serve any purpose; it's a partial QMK Configurator export, but it can't be used for anything in this format. It can and should be deleted:

```
git rm keyboards/keebio/bfo9000/keymaps/vladkvit/layers.json
```